src/JustSaying/Fluent/ISubscriptionBuilder`1.cs

```diff
@@ -20,24 +20,36 @@ public interface ISubscriptionBuilder<out T>
         /// Configures the middleware pipeline for this subscription.
         /// Any middleware configured here will be wrapped around a handler and metrics middleware.
         /// </summary>
-        /// <param name="middlewareConfiguration"></param>
+        /// <param name="middlewareConfiguration">A configuration action that provides an API
+        /// to override the default middleware behaviour. By default, <see cref="HandlerMiddlewareBuilderExtensions.UseDefaults{TMessage}"/> applies a set of
+        /// default middlewares to add metrics, error handling, completion handling, context setting, and logging.
+        /// Below is an example of how to add your own middlewares around the defaults.</param>
```

Comment:
Not sure if "Below..." would make sense when viewed in an Intellisense context.

Reply:
Yeah fair - at least on rider it's not particularly useful. I think the example is clear enough, no need to restate it.

![image](https://user-images.githubusercontent.com/130231/133334740-afd4ba59-9f8a-4e51-ac7d-589e1eaa7cb3.png)